src/lib/loaders/api/loader_without_authentication_factory.ts

```diff
@@ -127,29 +144,24 @@ export const apiLoaderWithoutAuthenticationFactory = <T = any>(
               return (
                 cache
                   .get(key)
-                  // Cache hit
-                  .then(
-                    finish({
-                      message: `Cached: ${key}`,
-                      cached: true,
-                    })
-                  )
-                  // Cache miss.
-                  .catch(() => {
-                    const cacheOptions: CacheOptions = {}
-                    if (apiOptions.requestThrottleMs) {
-                      cacheOptions.cacheTtlInSeconds =
-                        apiOptions.requestThrottleMs / 1000
+                  // Successful cache operation.
+                  .then((data) => {
+                    if (data !== undefined) {
+                      // Cache hit.
+                      return finish({
+                        message: `Cached: ${key}`,
+                        cached: true,
+                      })(data)
                     }
-                    return callApi()
-                      .then(
-                        finish({
-                          message: `Requested (Uncached): ${key}`,
-                          cached: false,
-                        })
-                      )
-                      .then(reduceData)
-                      .then((data) => cacheData(data, cacheOptions))
+
+                    // Cache miss.
+                    return fetchAndCache()
+                  })
+                  // Cache error.
+                  .catch((err) => {
+                    // Log the error and continue with the request.
+                    error(err)
```

Comment:
This line is new - before we weren't able to distinguish between a cache error and a miss, and so we would always just fall back to the API. Now, we know this is an actual cache operation error of some kind, so we can log it.